pkg/cmd/cluster/connect.go

```diff
@@ -74,6 +74,10 @@ var connectExample = templates.Examples(`
 
 const passwordMask = "******"
 
+var kbcliEnginesBlackList = []string{
+	string(models.PolarDBX),
```

Comment:
https://github.com/apecloud/kbcli/issues/180  kbcli do not support connect to starrocks, may be add it to this list.